eng/platform-matrix.yml

```diff
@@ -160,6 +160,38 @@ jobs:
         - RedHat.7.Amd64
       ${{ insert }}: ${{ parameters.jobParameters }}
 
+# Linux gcc x64
+
+- ${{ if or(containsValue(parameters.platforms, 'Linux_gcc_x64'), in(parameters.platformGroup, 'all', 'gcstress')) }}:
+  - template: ${{ parameters.jobTemplate }}
+    parameters:
+      stagedBuild: ${{ parameters.stagedBuild }}
+      buildConfig: ${{ parameters.buildConfig }}
+      archType: x64
+      osGroup: Linux
+      osSubgroup: _gcc
+      managedTestBuildOsGroup: OSX
+      container:
+        image: centos-7-f39df28-20191023143754
+        registry: mcr
+      helixQueues:
+      - ${{ if and(eq(variables['System.TeamProject'], 'public'), in(parameters.helixQueueGroup, 'pr', 'ci', 'corefx')) }}:
+        - Ubuntu.1804.Amd64.Open
```

Comment:
If we end up running testing for gcc build. I would suggest only running one platform. The full set is generally expensive.